### ISSUE TITLE
Updating load-balancer-controller CRDs is failing to find the repository

### ISSUE THREAD

**Issue by robert-hunter-nes** [aws/eks-charts#1033]
**Describe the bug**
Running the command to update the CRDs is returning an error saying that it can't find the repository.

I was using the command [here](https://github.com/aws/eks-charts/blob/master/stable/aws-load-balancer-controller/README.md#upgrading-from--aws-load-balancer-controller-v213-and-earlier).

**Steps to reproduce**
The command I am running and the results are below
``` shell
$ kubectl apply -k "github.com/aws/eks-charts/stable/aws-load-balancer-controller//crds?ref=v0.0.154"
error: failed to run '/opt/homebrew/bin/git fetch --depth=1 https://github.com/aws/eks-charts/stable/aws-load-balancer-controller v0.0.154': remote: Not Found
fatal: repository 'https://github.com/aws/eks-charts/stable/aws-load-balancer-controller/' not found
: exit status 128
```

**Expected outcome**
The charts should update successfully.

**Environment**

* Chart name: aws-load-balancer-controller
* Chart version: v0.0.154
* Kubernetes version: 1.27
* Using EKS (yes/no), if so version? v1.27.7-eks-4f4795d

**Additional Context**:
This has worked before and stopped working on Wednesday morning (29/11).

**Comment by robert-hunter-nes** [comment]
Just realised it seems to be because of the extra forward slash before "crds" in the address, the below works OK.

``` shell
$ kubectl apply -k "github.com/aws/eks-charts/stable/aws-load-balancer-controller/crds?ref=v0.0.154"
```

**Comment by bryantbiggs** [comment]
resolved in #1048
